src/main.c

```diff
@@ -147,7 +147,7 @@ void ShowArguments(void) {
 }
 
 // NOT IMPLEMENTED
-int Savetofile(char* pass, char* filename) {
+int SaveToFile(char* pass, char* filename) {
 	FILE* file = fopen(filename, "w");
 
 	if (file != NULL) {
```

Comment:
What do you mean as long as it does not contain spaces or newlines?
I considered spaces as special characters.
Also would this would not work since we have to append a NULL character to the end of the string, this will continue to print random stuff until it reaches a NULL character

Reply:
From the Library Functions Manual regarding s specifier:

If no l modifier is present: the const char * argument is expected to be a pointer to an array of character type (pointer to a
              string).  Characters from the array are written up to (but not including) a terminating null byte ('\0'); if  a  precision  is
              specified,  no more than the number specified are written.  If a precision is given, no null byte need be present; if the pre‐
              cision is not specified, or is greater than the size of the array, the array must contain a terminating null byte.


I think the entire ' ' or '\n' thing comes from the scanf family of functions.

btw I'm realising I should maybe be using sscaf instead of atoi, it's more appropriate for strings

sscanf manual:

The format string consists of a sequence of directives which describe how to process the sequence of input characters.  If processing of  a  directive  fails,  no
       further input is read, and sscanf() returns.  A "failure" can be either of the following: input failure, meaning that input characters were unavailable, or match‐
       ing failure, meaning that the input was inappropriate (see below).

       A directive is one of the following:

       •      A sequence of white-space characters (space, tab, newline, etc.; see isspace(3)).  This directive matches any amount of white space, including none, in the
              input.

       •      An ordinary character (i.e., one other than white space or '%').  This character must exactly match the next character of input.

       •      A  conversion specification, which commences with a '%' (percent) character.  A sequence of characters from the input is converted according to this speci‐
              fication, and the result is placed in the corresponding pointer argument.  If the next item of input does not match the conversion specification, the  con‐
              version fails—this is a matching failure.